### ISSUE TITLE
Seeding overpowering low performance WMS/WMTS server

### ISSUE THREAD

**Issue by sanderpukk** [mapproxy/mapproxy#529]
## Context

We have to seed from an existing WMTS service provider into our own MapProxy and Mapserver system because our services need to be 24/7 and high capability. 
While we seed from them, MapProxy basically overwhelms them and there isnt that much we can do about it besides getting hard copies of the needed tiles.

## Possible Fix

Already using it from 1.13 version. Have already added to our process, but dont mind sharing.
Basically add a  parameter for the seeding process where the user can set how many times does the seeding request a tile when no response is given or error. Then another parameter for adding wait time between request so the seeding can spread out.

Overall adds two variables to control the seeding process more. In default mode nothing changes.

**Comment by weskamm** [comment]
Did you already try the `concurrent_requests` parameter?

https://mapproxy.org/docs/latest/sources.html#wms-source-concurrent-requests-label

Why does it not solve the problem?

**Comment by sanderpukk** [comment]
concurrent is 1. It's still too much most of the time. And it doesnt solve the max_request anyways.

**Comment by sanderpukk** [comment]
I would like to do a pull request to add that parameter. It does not change default behavior and is useful for bad WMS and WMTS service providers.
